raiden-dapp/src/components/stepper/StepperDivider.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-divider
+    class="stepper-divider"
+    :class="{
+      'stepper-divider--active': active,
+      'stepper-divider--skipped': skipped,
+    }"
+  />
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+@Component
+export default class StepperStep extends Vue {
```

Comment:
`class StepperDivider extends...` not `StepperStep`

Reply:
✔️